server_py/flatgov/templates/bills/detail.html

```diff
@@ -297,13 +301,48 @@ <h4>Sponsors of Related Measures</h4>
         <li class="nav-item" role="presentation">
             <a class="nav-link active" id="all-congress-sponsor-tab" data-toggle="tab" href="#all-congress-sponsor" role="tab" aria-controls="all-congress-sponsor" aria-selected="true">All Congresses</a>
         </li>
+        <li class="nav-item" role="presentation">
```

Comment:
"This Congresses" should be "Current Congress"; also, I'm not sure why it is being shown in a tab after "All Congresses". I don't see this reordering in Daniel's feedback.